packages/@lightningjs/ui-components/src/components/InlineContent/InlineContent.js

```diff
@@ -508,6 +529,9 @@ export default class InlineContent extends Base {
           announce += item.announce;
         } else if (item.text) {
           announce += item.text;
+          if (item.style?.textDecoration === 'line-through') {
+            announce += 'strikethrough';
+          }
```

Comment:
just checking, is override-able? @ImCoolNowRight i vaguely remember this coming up during our last sync. did we decide that users should be able to pass whatever announce text they want into this like: "Price used to be __ but is now __".

Reply:
Still waiting on confirmation from the accessibility team. Personally, I think this is a good generic default, and that clients can override `_announce` for their needs on a case by case basis. But we'll see what they say!

Reply:
Resolving this conversation to get this merged in. Will create a follow-up ticket to ensure this gets completed.